tests/kvbm_integration/test_consolidator_router_e2e.py

```
@@ -759,15 +867,60 @@ def test_remove_deduplication_across_sources(
+                from concurrent.futures import ThreadPoolExecutor, as_completed
+
+                def send_request(request_idx: int) -> tuple[int, bool]:
+                    """Send a single request and return (index, success)."""
+                    prompt = f"Tell me a unique story about topic {request_idx}. Make it very long and detailed with many paragraphs."
+                    try:
+                        response = tester.send_chat_request(
+                            messages=[{"role": "user", "content": prompt}],
+                            max_tokens=max_tokens,
+                        )
+                        success = "content" in response["choices"][0]["message"]
+                        return (request_idx, success)
```

Ruff (0.14.10)

Consider moving this statement to an `else` block

(TRY300)